src/test/bin/quic_gtest.cpp

```diff
@@ -214,6 +214,42 @@ TEST(ParameterValidation, ValidateConnection) {
     }
 }
 
+TEST(OwnershipValidation, RegistrationShutdownBeforeConnOpen) {
+    TestLogger Logger("RegistrationShutdownBeforeConnOpen");
+    if (TestingKernelMode) {
+        //ASSERT_TRUE(DriverClient.Run(IOCTL_QUIC_RUN_VALIDATE_API));
```

Comment:
You plan on doing kernel too?

Reply:
Yes. I'm working on a few more tests too.